plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/PostgreUtils.java

```diff
@@ -126,6 +126,7 @@ public static boolean isPGObject(Object object) {
         }
         String className = object.getClass().getName();
         return className.equals(PostgreConstants.PG_OBJECT_CLASS) ||
+			className.equals(PostgreConstants.GAUSSDB_PG_OBJECT_CLASS) ||
```

Comment:
Please keep 4-spaces indentation that we use
```suggestion
        className.equals(PostgreConstants.GAUSSDB_PG_OBJECT_CLASS) ||
```

Reply:
Thank you very much for your suggestion. It has been revised.